types/textfit/textfit-tests.ts

```diff
@@ -14,3 +15,15 @@ const options: textFit.TextFitOption = {
 };
 
 textFit(element, options);
+textFit(elements, options);
+
+const ul = document.createElement('ul');
+ul.append(document.createElement('li'));
+
+const nodeList = ul.querySelectorAll('li');
```

Comment:
I'm not really confident that `NodeList` is supported, I wasn't able to find this as an example in the library, though I may have overlooked it. Where did you see it referenced?

Reply:
I referenced [`index.d.ts` of textFit repository](https://github.com/STRML/textFit/commit/5f8a94#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8R20). 

This package already have own types a year ago, but is still work-in-progress status. 

please refer https://github.com/STRML/textFit/issues/60 issue.